src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/Teams/TeamNameViewModel.cs

```diff
@@ -70,5 +91,23 @@ public Team ToDomain()
                 Name = Name
             };
         }
+
+        #region private
+
+        /// <summary>
+        /// Returns groups list in the division
+        /// </summary>
+        /// <param name="team">Mapped team</param>
+        /// <param name="divisions">Divisions list in the current tournament</param>
+        /// <returns>List of type Group</returns>
+        private static List<Group> GetGroups(TeamNameViewModel team, List<Division> divisions)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Consider using more general type 'System.Collections.Generic.IEnumerable<VolleyManagement.Domain.TournamentsAggregate.Division>' instead of 'System.Collections.Generic.List<VolleyManagement.Domain.TournamentsAggregate.Division>'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3242)